src/decimal.rs

```diff
@@ -548,7 +556,7 @@ impl Decimal {
     }
 
     #[must_use]
-    pub(crate) const fn from_parts_raw(lo: u32, mid: u32, hi: u32, flags: u32) -> Decimal {
+    pub const fn from_parts_raw(lo: u32, mid: u32, hi: u32, flags: u32) -> Decimal {
```

Comment:
Was there a reason for changing the visibility of this? The reason it was kept this way as it allows flags to get into an invalid state. `from_parts` is similar, however ensures internal state is kept valid.

Reply:
I agree it should be private; I probably forgot to change it after debugging.